lib/output.js

```diff
@@ -694,6 +694,7 @@ function OutputStream(options) {
         output.semicolon();
     });
     DEFPRINT(AST_Toplevel, function(self, output){
+        use_asm = false;
```

Comment:
as the object key quoting issue indicated, the problem may unrelated to the specific changes merged, but instead some pre-existing regression in harmony.